cni/network/invoker_cns.go

```diff
@@ -136,7 +137,6 @@ func (invoker *CNSIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, erro
 			}
 		} else {
 			logger.Info("Failed to get IP address from CNS",
-				zap.Error(err),
```

Comment:
why we remove logging err? isnt expectation to remove stack trace but log error

Reply:
I believe this error is wrapped and returned on 141, which should bubble up to Add's err variable. When we reach the defer in Add(), we will print out this err variable at that log statement. Leaving the log in 139 I believe would cause the error to be logged twice.

Reply:
ok.. then we should remove this log too.. either log with error or remove log line

Reply:
I initially kept this log since it logs the "response" variable on 140 unless we don't care about this info?